infra/build/developer-tools/build/install_cft_cli.sh

```diff
@@ -20,5 +20,5 @@ CFT_CLI_VERSION=$1
 
 cd /build
 
-wget "https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/releases/download/v${CFT_CLI_VERSION}/cft-linux-amd64"
+wget "https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/releases/download/cli%2Fv${CFT_CLI_VERSION}/cft-linux-amd64"
```

Comment:
This will fail for v0.5.0 since that is still at v0.5.0 and not cli/v0.5.0. Can you put an if/else to check if `wget` returns a non-zero, `wget` the previous URL else do the new one?

Reply:
Yes, I might need to fiddle with `set -e` as well.

Reply:
OK - It will now first check for the new format, and then fall back to the old if that errors.

Reply:
@g-awmalik - This should be good now.